src/hooks/useGetTooltipStyles.ts

```diff
@@ -5,6 +5,17 @@ import { useGetTooltipPosition } from './useGetTooltipPosition'
 
 const TOOLTIP_GAP = 8
 
+enum Positions {
+  bottomLeft = 'bottom-left',
+  bottomRight = 'bottom-right',
+  topLeft = 'top-left',
+  topRight = 'top-right',
+  middleLeft = 'middle-left',
+  middleRight = 'middle-right',
+  bottomCenter = 'bottom-center',
+  topCenter = 'top-center'
+}
+
```

Comment:
```suggestion
enum Positions {
  BOTTOM_LEFT = 'bottom-left',
  BOTTOM_RIGHT = 'bottom-right',
  TOP_LEFT = 'top-left',
  TOP_RIGHT = 'top-right',
  MIDDLE_LEFT = 'middle-left',
  MIDDLE_RIGHT = 'middle-right',
  BOTTOM_CENTER = 'bottom-center',
  TOP_CENTER = 'top-center'
}

```

Because enum kays is constants

Reply:
also please change TTooltipProps interface's position prop to this enum